src/screens/ProfileScreen/styles.js

```diff
@@ -0,0 +1,18 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+    LoginButtonView : {
+        width: "100%", 
+        height: 50, 
+        backgroundColor: "#51AD28", 
```

Comment:
If these color codes are global, which means repeated in different files, put them into a single file and import them. In this way app theme can be easily modify.

Reply:
Done!